### PR TITLE
Migrate to Docusaurus

### PR DESCRIPTION
**Pull request by significance**

### CODE REVIEW

**Review by Eknir — APPROVED** [review]
Thanks for this initial commit. Work to do to clean up Docusaurus, but the good way forward!

File: docs/installation/simple-node.md [hunk 1]
```diff
@@ -1,8 +1,5 @@
 ---
-title: 'Installation'
-weight: 2
-summary: "How to install client code to run a node."
-alias: "/bee-docs/installation.html"
+title: Installation
```
Comment (Eknir) [inline docs/installation/simple-node.md:2]: no `id`?

File: blog/2019-05-28-hola.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (Eknir) [inline blog/2019-05-28-hola.md:1]: We can remove everything related to blogs, right?

File: docusaurus.config.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
+module.exports = {
+  title: 'Swarm Bee Documentation',
+  tagline: 'Welcome to the swarm',
+  url: 'https://your-docusaurus-test-site.com',
```
Comment (Eknir) [inline docusaurus.config.js:4]: Link to our github?

File: docusaurus.config.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        src: 'img/swarm-logo.svg',
+      },
+      items: [
+        {
+          to: 'docs/',
+          activeBasePath: 'docs',
+          label: 'Docs',
+          position: 'left',
+        },
+        {to: 'blog', label: 'Blog', position: 'left'},
```
Comment (Eknir) [inline docusaurus.config.js:24]: remove?

File: docusaurus.config.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+          title: 'Docs',
+          items: [
+            {
+              label: 'Swarm',
+              to: 'https://ethswarm.org',
+            }
+          ],
+        },
+        {
+          title: 'Community',
```
Comment (Eknir) [inline docusaurus.config.js:45]: Link: Mattermost (Beehive), Medium, Twitter, Telegram

File: docusaurus.config.js [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+              href: 'https://discordapp.com/invite/docusaurus',
+            },
+            {
+              label: 'Twitter',
+              href: 'https://twitter.com/docusaurus',
+            },
+          ],
+        },
+        {
+          title: 'More',
```
Comment (Eknir) [inline docusaurus.config.js:62]: remove?

File: docusaurus.config.js [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+      {
+        docs: {
+          // It is recommended to set document id as docs home page (`docs/` path).
+          homePageId: 'introduction',
+          sidebarPath: require.resolve('./sidebars.js'),
+          // Please change this to your repo.
+          editUrl:
+            'https://github.com/facebook/docusaurus/edit/master/website/',
+        },
+        blog: {
```
Comment (Eknir) [inline docusaurus.config.js:90]: remove?

**Review by Eknir — COMMENTED** [review]
Left some other notes, mostly to do a clean up. For me, we can merge without this clean up and iterate on it. However, if you can do some more clean up before merge, without loosing too much time it would be good to do!

File: blog/2019-05-28-hola.md (hunk 2 above)
Comment (significance) [reply blog/2019-05-28-hola.md:1]: i would like to use it as a releaselog with a short paragraph explaining what has been released and any breaking changes

there are a bunch of other bee-based technical writings eg. monthly dev update, devops comms and so on that could  also be included here depending on the functionality available, certainly seems a shame to waste it but let's keep it simple for initial release

**Review by vandot — APPROVED** [review]

**Comment by vandot** [comment]
All that @Eknir mentioned

**Comment by significance** [comment]
Merging now as i'm shifting from infra to content 🤙

**Pull request merged**